README.md

```diff
@@ -1 +1,232 @@
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+
+For more information on the importance of a professional README for your repositories: https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/readme_best_practices.md
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+ 
+  <br/>
+
+  <h3><b>Todo List</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+<a name="about-project"></a>
+
+   
+
+## 🛠 Built With  <a name="built-with"></a>
+1- HTML.
+2- CSS.
+3- JAVASCRIPT
+3- LINTERS.
+4- WEBPACK
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">JAVASCRIPT</a></li>
+    <li><a href="https://reactjs.org/">WEBPACK</a></li>
+    
+  </ul>
+</details>
+
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- Added index.html file.
+- Added style.css file.
+- Added javascript files.
+- Adding a new item.
+- Removing a selected item.
+- Marking a selected item as complete.
+- Removing all items marked as complete at once
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+> Comgit ing Soon
+<!-- Click [live demo link to view project](https://jimoluwaseyi.github.io/module-one-capstone/) -->
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+ 1- Web browser.
+ 2- Code editor.
+ 3- Git-SCM.
+ 4- npm run dev
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Run this commands:
+
+```sh
+  cd webpack-project
+  git clone https://github.com/JimOluwaseyi/Webpack-project.git
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+ Open index.html in your text editor and run using live server extension.
+
```

Comment:
- [ ] Kindly include that `npm install` should be run after cloning the project and `npm run dev` should be used to start the project.